### DISCUSSION TITLE
> Error: Spawn failed at ChildProcess. (/data/data/com.termux/files/home/blog/node_modules/hexo-util/lib/spawn.js:51:21) at ChildProcess.emit (node:events:513:28) at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12) emm不知道为什么

### DISCUSSION THREAD

**Discussion by Dereck-327** [hexojs/hexo#5498] [Question / Answer]
> Error: Spawn failed at ChildProcess. (/data/data/com.termux/files/home/blog/node_modules/hexo-util/lib/spawn.js:51:21) at ChildProcess.emit (node:events:513:28) at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12) emm不知道为什么

可以试试用手机给电脑开热点

_Originally posted by @namaemonaishi in https://github.com/hexojs/hexo-deployer-git/issues/193#issuecomment-1729363780_
   我关闭代理了也还是不行欸

**Comment by uiolee** [comment]
Please provide complete log
